test/staking/TribalChief.test.js

```diff
@@ -213,7 +215,21 @@ describe('TribalChief', () => {
       this.tribe = await Tribe.new();
       this.coreRef = await MockCoreRef.new(this.core.address);
 
-      this.tribalChief = await TribalChief.new(this.core.address, this.tribe.address);
+      /// we can and probably should do this with a framework,
+      // but for now I'm going to write it out by hand.
+
+      // spin up the logic contract by hand
+      const tribalChief = await TribalChief.new(this.core.address);
+      // create a new proxy contract
+      const proxyContract = await Proxy.new(tribalChief.address, tribalChief.address, '0x', { from: userAddress });
```

Comment:
We should build the calldata to initialize things in this tx when creating this new proxy.

Reply:
I think its fine as-is for the unit tests. We do the atomic calldata init on the main deploy and e2e tests